addons/ctrl-click-run-scripts/addon.json

```diff
@@ -25,7 +23,43 @@
       "matches": ["projects"]
     }
   ],
+  "settings": [
+    {
+      "name": "Run by...",
+      "id": "run-action",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "single",
+          "name": "Clicking once"
+        },
+        {
+          "id": "double",
+          "name": "Clicking twice"
+        },
+        {
+          "id": "ctrl",
+          "name": "Ctrl-clicking"
+        }
+      ],
+      "default": "single"
```

Comment:
I'm not sure how to default this setting to `"ctrl"` for existing addon users who enabled this addon. The `chrome.runtime.onInstalled` event's details have a `previousVersion` property which we could leverage, but I'm not quite sure how to work it into our setting migration procedure.

We could just make the default option `"ctrl"` for the next release and then change it to `"single"` again for the release afterward. That way, the behavior doesn't change for existing users of this addon (but also for existing non-users).